vdom/create-element.js

```diff
@@ -26,6 +26,7 @@ function createElement(vnode, opts) {
         return null
     }
 
+    var tagName = vnode.tagName || "span"
```

Comment:
It has no relevance. At the moment I’m just checking out what I need to implement to make my lib compatible with `virtual-dom`. So let’s make it a `div` :)